module/tools/SystemConfiguration/data/system/default/etc/hostname

```diff
@@ -0,0 +1 @@
+nubots
```

Comment:
this is like the one file that should be different per device!

Reply:
damn, missed that one lol